int32     platform_id
Point2d[] path_points
bool      start_fine_tune_mode
bool      start_connection
